From cb82f3d65c200aac959f461f1004e18ace851eb9 Mon Sep 17 00:00:00 2001
From: OTS <octavian-tiberiu.sirbu@nxp.com>
Date: Fri, 29 Aug 2025 11:48:04 +0300
Subject: [PATCH] Removed PYTHON_INCLUDE_DIRS from CMakeLists

Signed-off-by: OTS <octavian-tiberiu.sirbu@nxp.com>
Upstream-Status: Pending
---
 CMakeLists.txt | 1 -
 1 file changed, 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index c0ef675..fdf980b 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -45,7 +45,6 @@ add_library(${PROJECT_NAME} INTERFACE)
 
 target_include_directories(${PROJECT_NAME}
                            INTERFACE
-                           ${PYTHON_INCLUDE_DIRS}
                            $<BUILD_INTERFACE:${PYBIND11_JSON_INCLUDE_DIR}>
                            $<INSTALL_INTERFACE:include>)
 
-- 
2.34.1

